src/field_theory/intermediate_field.lean

```diff
@@ -271,6 +271,8 @@ def map (f : L →ₐ[K] L') : intermediate_field K L' :=
   neg_mem' := λ x hx, (S.to_subalgebra.map f).neg_mem hx,
   .. S.to_subalgebra.map f}
 
+@[simp] lemma coe_map (f : L →ₐ[K] L') : ↥(S.map f) = ↥(f '' S) := rfl
```

Comment:
This is an equality of types, which is a bit uncomfortable; would
```
@[simp] lemma coe_map (f : L →ₐ[K] L') : (S.map f : set L') = (f '' S) := rfl
```
be better?

Reply:
Yael seems to think it's okay, so I've left it as is

Reply:
I've removed this since the proof doesn't use them, and @YaelDillies has PR'd it separately as #15188